votor/src/event_handler.rs

```diff
@@ -684,7 +687,10 @@ impl EventHandler {
         // it's not necessary to vote skip prior to it and we won't
         // be able to check vote history if we've already voted on it
         let root_bank = voting_context.root_bank.load();
-        let start = first_of_consecutive_leader_slots(slot).max(root_bank.slot());
+        // No matter what happens, we should not vote skip for slot 0
```

Comment:
why not mark slot 0 as having a Notar vote from us? at the same time this would get rid of the special case for slot 1 in `try_notar()`

Reply:
Hmm, I think we currently require a ParentReady on slot 1, I guess the point is to make sure everyone starts at roughly the same time.
But in reality the leader of slot 1 almost never makes it, because of our rolling restart model. So normally in rolling restart model the earlier restarted validators just keep sending skip 1 to 3 waiting for a skip certificate.
Are you suggesting that instead of special casing requiring a ParentReady on slot 1 in try_notar, on start up we actually send Notarize block 0 (even though it's already a root) to start the timer?
Or we should not really send out Notarize block 0 but special case 0 to start the timer?

Reply:
Yeah i think that makes sense, when creating a fresh VoteHistory at genesis, we can additionally mark that we have voted notarize. 

For now this is fine, and we can adjust in a follow up